tests/unit/cluster/failover2.tcl

```diff
@@ -64,3 +64,34 @@ start_cluster 3 4 {tags {external:skip cluster} overrides {cluster-ping-interval
     }
 
 } ;# start_cluster
+
```

Comment:
This test may be time-consuming. It basically cannot pass before the patch, but can pass locally after the patch.